GhostRaiderII/Offsets.h

```diff
@@ -9,4 +9,18 @@ namespace Offsets {
 		static constexpr uintptr_t renderLaraFunc = 0x419DD0;
 		static constexpr uintptr_t title = 0x11BD90;
 	};
+	
+	class Steam_UK {
+	public:
+		static constexpr uintptr_t loadLevelFunc = 0x43C780;
+		static constexpr uintptr_t renderLaraFunc = 0x419DF0;
+		static constexpr uintptr_t title = 0x11BD90;
```

Comment:
`title` should be the same across versions. We should refactor it out of the classes!